db/migrate/20200428110354_add_index_to_test_results.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToTestResults < ActiveRecord::Migration[5.2]
+  def change
+    add_index :test_results, [:profile_id, :host_id, :end_time]
```

Comment:
The only missing index should be `end_time`. You can check in db/schema.rb:

```
  create_table "test_results", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
    t.datetime "start_time"
    t.datetime "end_time"
    t.decimal "score"
    t.uuid "profile_id"
    t.uuid "host_id"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.index ["host_id"], name: "index_test_results_on_host_id"
    t.index ["profile_id"], name: "index_test_results_on_profile_id"
  end
```

Reply:
This is not adding 3 indices - it's adding a combined index on the 3 fields which is used everytime we call `latest` https://github.com/RedHatInsights/compliance-backend/blob/master/app/models/test_result.rb#L16

Reply:
Ok, I see - this should be `unique: true` as I've done here: https://github.com/RedHatInsights/compliance-backend/pull/406/files#diff-277bf976622928a3a96e1377f1359bb6R5

Otherwise, we will need to remove this index and re-add it as unique when #406 is merged.